### PR TITLE
fix: JSON is missing actual search results (#56)

### PR DESCRIPTION
**Pull request by hawkeye64**
Fixes hierarchical data by adding recursive method. Also adding two additional Jest tests

### CODE REVIEW

**Comment by hawkeye64** [comment]
Looks like I need to lint it...

File: src/googleIt.js [hunk 1]
```diff
@@ -36,14 +36,23 @@ export function openInBrowser(open, results) {
...
-    .map((child) => {
-      if (!child.data) {
-        return child.children.map((c) => c.data);
-      }
-      return child.data;
-    })
-    .join('');
+  // recursive function to get "all" the returned data from Google
+  function findData(child) {
+    if (!child.data) {
```
Comment (IlCallo) [inline src/googleIt.js:41]: Maybe a bit cleaner `return c.data || findData(c)`?

Comment (hawkeye64) [reply src/googleIt.js:41]: @IlCallo  I've had to make changes because of rstrictiove eslint - please review update if you can. Thanks.

Comment (hawkeye64) [reply src/googleIt.js:41]: Sorry, should have read that beeter. Yes, you are right.

Comment (hawkeye64) [reply src/googleIt.js:41]: @IlCallo You should see the requested changes now. I always appreciate your reviews. Thanks.

**Comment by hawkeye64** [comment]
Also added two new tests. All passing:
```
Test Suites: 4 passed, 4 total
Tests:       38 passed, 38 total
Snapshots:   0 total
Time:        3.451 s
Ran all test suites.
Done in 4.16s.
```

**Review by PatNeedham — APPROVED** [review]

**Pull request merged**